contracts/managers/fee/FeeManager.sol

```diff
@@ -544,6 +567,52 @@ contract FeeManager is AccessManagedUpgradeable, ContextClient {
         return (PERFORMANCE_FEE_ACCOUNT, feeShares);
     }
 
+    /// @notice Calculates the deposit fee for a given amount of assets
+    /// @dev This function calculates the deposit fee based on the configured deposit fee percentage
+    ///      and the amount of assets being deposited. The deposit fee is calculated as a percentage
+    ///      of the deposited assets using 18 decimal precision (1e18 = 100%, 1e16 = 1%).
+    ///
+    /// @param assets_ The amount of assets being deposited (in asset units)
+    ///
+    /// @return The calculated deposit fee amount (in asset units)
+    ///
+    /// @dev Flow:
+    /// 1. Retrieves the current deposit fee percentage from storage
+    /// 2. If deposit fee is 0, returns 0 (no fee)
+    /// 3. Calculates fee using Math.mulDiv for precision: (assets_ * depositFee) / 1e18
+    ///
+    /// @custom:security Uses Math.mulDiv to prevent overflow and maintain precision
+    /// @custom:security View function - no state changes, safe for external calls
+    /// @custom:see EIP-4626 for vault fee patterns and best practices
+    /// @custom:example If depositFee is 1e16 (1%) and assets_ is 1000e18, returns 10e18
+    function calculateDepositFee(uint256 assets_) external view returns (uint256) {
+        uint256 depositFee = FeeManagerStorageLib.getPlasmaVaultDepositFee();
+        if (depositFee == 0) {
+            return 0;
+        }
+        return Math.mulDiv(assets_, depositFee, 1e18);
```

Comment:
returned value is not in 18 decimals but in decimals of "assets_" variable, please fix  description for this method